AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -600,6 +608,43 @@ open class DeckPicker :
                 else -> error("Unexpected fragment result key! Did you forget to update DeckPicker?")
             }
         }
+
+        CompatHelper.compat.configureView(
+            this,
+            pullToSyncWrapper,
+            IMPORT_MIME_TYPES,
+            DropHelper.Options.Builder()
+                .setHighlightColor(R.color.material_lime_green_A700)
+                .setHighlightCornerRadiusPx(0)
+                .build(),
+            onReceiveContentListener
+        )
+    }
+
+    private val onReceiveContentListener = OnReceiveContentListener { _, payload ->
+        val (uriContent, remaining) = payload.partition { item -> item.uri != null }
+
+        if (uriContent == null) {
+            return@OnReceiveContentListener remaining
+        }
+
+        val clip = uriContent.clip
```

Comment:
I don't know if I like this better
```suggestion
        val clip = uriContent?.clip ?: return@OnReceiveContentListener remaining
```